scania-digital-design-system/tegel
# build: stencil upgrade, playwright image upgrade, tooltip test improvement

## **Describe pull-request**  
Upgrading stencil to newer version
Upgrading playwright docker image to newer version
Fixing tooltip test

## **Issue Linking:**  
- **No issue:** General maintenance of system

## **How to test**  
1. Open branch on your local
2. Drop `node_modules` in `packages/core``
3. Run `npm i` in `packages/core`
4. Try to run system by `npm run start` on root level
5. Try to run test by `npm run test` on root level
6. Everything should work as before

## **Checklist before submission**
